build.sbt

```diff
@@ -1,18 +1,22 @@
+
 import uk.gov.hmrc.DefaultBuildSettings
 
```

Comment:
In this file, please also remove or update the following:

- Remove [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/build.sbt#L22-L24) as it is redundant
- Remove [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/build.sbt#L30) when migrating away from the deprecated `IntegrationTest` scope
- Update [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/build.sbt#L32-L33) when when migrating away from the deprecated `IntegrationTest` scope
- Update [here](https://github.com/hmrc/tax-history/blob/DDCE-5261/run_all_tests.sh#L3) when when migrating away from the deprecated `IntegrationTest` scope